pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -128,30 +131,6 @@ def continue_single_sweep(self):
         self.write("INIT:CONM; *WAI")
 
     # Traces ------------------------------------------------------------------
-    '''
-    def read_trace(self, n_trace=1):
-
-        """
-        Read trace data.
-
-        :param n_trace: The trace number (1-6). Default is 1.
-        :return: 2d numpy array of the trace data, [[frequency], [amplitude]].
-        """
-
-       
-        dict = self.available_channels
-        if self.active_channel == ("PNO") or dict.get(self.active_channel) == "PNOISE":
-
-            y = np.array(self.values(f"TRAC? TRACE{n_trace}")[1::2])
-            x = np.array(self.values(f"TRAC? TRACE{n_trace}")[0::2])
-
-        elif self.active_channel == ("SAN") or dict.get(self.active_channel) == "SANALYZER":
-
-            y = np.array(self.values(f"TRAC? TRACE{n_trace}"))
-            x = np.linspace(self.freq_start, self.freq_stop, len(y))
-
-        return np.array([x, y])
-'''
```

Comment:
Please just delete outdated stuff right away instead of commenting them out, makes the diff's more readable.

Reply:
Resolved via [9d73b1d](https://github.com/bleykauf/pymeasure/pull/4/commits/9d73b1d02c1df0a713dcb7e6d7fef6921f07e2e9)